lib/modules/dosomething/dosomething_reportback/dosomething_reportback.install

```diff
@@ -92,9 +92,13 @@ function dosomething_reportback_schema() {
       ),
     ),
     'primary key' => array('rbid'),
+    'unique keys' => array(
+      'rbid-uid-nid-run_nid' => array('rbid', 'uid', 'nid', 'run_nid'),
```

Comment:
from slack
> 
```
select rb.rbid, rb.uid, rb.nid, count(*)
from dosomething_reportback rb
group by rb.nid, rb.uid
having count(*) > 1;
```

this proves there are multiple reportbacks with the combo of same nid/uid in prod/thor currently

so we can't have these be unique (at least not yet)